### PR TITLE
hotfix - switching over to [systemd] for [ir-server]

### PR DESCRIPTION
**Pull request by RobertLucian**
I needed to change the way `Advanced Communications Options` worked.

Before this PR, `Advanced Communications Options` was disabling the `ir server` service by removing a config file for the `monit` package.

With this PR, `Advanced Communications Options` now **disables/enables/starts/stops** the `ir server` through `systemctl`.

 ---------------

- [x] Test this PR on a fresh `Jessie` image.

----------------

Link related to the other PR on `Raspbian for Robots` : https://github.com/DexterInd/Raspbian_For_Robots/pull/141
Link related to the PR on `GoPiGo` : https://github.com/DexterInd/GoPiGo/pull/245

### CODE REVIEW

**Comment by RobertLucian** [comment]
@DexterInd Hey John, what do you think?

`Enable Bluetooth` icon doesn't appear in `Advanced Communication Options`. 

I've looked over the code and I don't see why the `bluetooth` label doesn't appear - there should be something with the `GUI` instead - maybe something that haven't got installed?

So, this have happened after I've updated the image with `R4R`. Also, I've had to modify `update_all.sh` and `fetch_gopigo.sh` scripts in order for them to update with my branches - so maybe something got wrong along the way. 
I'm not sure either if there are dependencies on which `Advanced Communication Options` depends on.

And this didn't happen yesterday.

---------

advanced_comm_nobt

**Comment by CleoQc** [comment]
are you on a Pi2 ?  It should only appear on a Pi3. It does on mine

**Comment by RobertLucian** [comment]
@CleoQc With my heart between the teeth, I hoped it was a `Pi2`. Unfortunately it's a `Pi3`.

**Comment by CleoQc** [comment]
https://github.com/DexterInd/Raspbian_For_Robots/blob/master/advanced_communication_options/advanced_communication_options.py#L47-L51  this is where the check is done

**Comment by CleoQc** [comment]
https://github.com/DexterInd/Raspbian_For_Robots/blob/master/advanced_communication_options/update_comms_settings.py#L118

**Comment by RobertLucian** [comment]
I've opened up a `python` interpreter, imported the said library and ran `check_pi3()`. 
I got a `False`.

**Comment by CleoQc** [comment]
If I compare to the latest Adafruit Pi Detection code, the way we detect is no longer valid? https://github.com/DexterInd/DI_Sensors/blob/master/DHT_Sensor/Adafruit_Python_DHT/Adafruit_DHT/platform_detect.py#L101-L106

**Comment by RobertLucian** [comment]
Let me try Adafruit's version and see what's giving me back.

**Comment by RobertLucian** [comment]
@CleoQc Thanks for showing me their version. 
This one seems to return the correct version on a `RPI 3`. 
I'm trying it now on a `RPI 2` too.

**Comment by CleoQc** [comment]
and I have a Pi3 and the Adafruit version returns it as a 2. At least that would explain why I was getting the BLuetooth option.  Did they change the Pi3 on us? I have an 'old' Pi3

File: advanced_communication_options/update_comms_settings.py [hunk 1]
```diff
@@ -125,8 +196,8 @@ def check_pi3():
 	if hw_id=="BCM2709":
```
Comment (RobertLucian) [inline advanced_communication_options/update_comms_settings.py]: @CleoQc  Actually, if you take a look here, you can see that `BCM2709` is actually the chip that `RPI 2` has.
Take a look in `Adafruit's` version and you'll see the difference.

Comment (RobertLucian) [inline advanced_communication_options/update_comms_settings.py]: Huh, so you're saying that the Raspberry Pi 3 has 2 chips depending on date that was produced?
Like it has the `BCM2709`/`BCM2835`?

Comment (RobertLucian) [inline advanced_communication_options/update_comms_settings.py]: Lol, this is what I get.

pi3_version

Comment (CleoQc) [inline advanced_communication_options/update_comms_settings.py]: Looks like it.  The Pi I have does have the bluetooth/wifi antenna on it and it's detected as a Pi2

**Comment by RobertLucian** [comment]
The issue of not seeing the 3rd label called `Enable Bluetooth` in `Advanced Communication Options` is going to be solved by PR : https://github.com/DexterInd/script_tools/pull/13

**Pull request merged**